src/constants/errorMessages.ts

```diff
@@ -1,5 +1,7 @@
 export const LOGIN_ERROR_MSG: string = '로그인에 실패했습니다.';
 export const LOGIN_ERROR_CHECK_MSG: string =
   '아이디와 비밀번호를 다시 확인해주세요';
+export const STATUS_MSG: string = '아직 사진이 생성되지 않았습니다.';
```

Comment:
사진 생성이 실패했다는 의미를 더욱 강조하기 위해서
`STATUS_MSG` 보다 
```suggestion
export const GENERATE_IMG_ERROR_MSG: string = '아직 사진이 생성되지 않았습니다.';
```
이런식으로 바꾸는 것은 어떨까요?
밑에 있는 것도 같이 바꿔 주시는 것을 제안드립니다

Reply:
밑에 `GENERATED_ERROR_MSG`가 있어서 `IMG_NOT_READY`로 바꾸고 밑에 있는거는 그 아래 `GENERATION_ERROR_MSG`랑 똑같아서 재활용했습니다